shell/packages/sandstorm-ui-topbar/topbar.js

```diff
@@ -44,6 +46,10 @@ Template.sandstormTopbarBlockReload.onDestroyed(function () {
   }
 });
 
+Template.sandstormTopbar.onRendered(function () {
+  this.drag = dragula([document.getElementById("navbar-grains")]);
```

Comment:
`onRendered()` gets called potentially many times for a single Template instance. Given that there's a cleanup hook in `onDestroyed`, does it make sense to check if `this.drag` is `undefined`, and if not, `destroy()` that instance before assigning the new one here?